core/modules/parsers/wikiparser/rules/syslink.js

```diff
@@ -0,0 +1,45 @@
+/*\
+title: $:/core/modules/parsers/wikiparser/rules/syslink.js
+type: application/javascript
+module-type: wikirule
+
+Wiki text inline rule for system tiddler links.
+Can be suppressed preceding them with `~`.
+\*/
+(function(){
+
+/*jslint node: true, browser: true */
+/*global $tw: false */
+"use strict";
+
+exports.name = "syslink";
+exports.types = {inline: true};
+
+exports.init = function(parser) {
+	this.parser = parser;
+	// Regexp to match
+	this.matchRegExp = /~?(?:\$):[^\s<>{}\[\]`|'"\\^~]+(?:\/|\b)/mg;
```

Comment:
> Anyhow: I think I do not fully understand what this is about

I think that @tobibeer is saying that when he experimentally removed angled brackets from the regexp for matching system tiddlers then he experienced a crash when viewing a system tiddler that had angle brackets in its title.